dvc/main.py

```diff
@@ -58,6 +58,9 @@ def main(argv=None):
         ret = 253
     except DvcParserError:
         ret = 254
+    except DvcException:
+        ret = 255
+        logger.exception("")
```

Comment:
DvcException won't have `unexpected error` prepended. Other exceptions and assertion errors will still have that prefixed, which is a good compromise, I'd say.